client/src/interface/Profile.ts

```diff
@@ -0,0 +1,22 @@
+export interface Profile {
+  _id: string;
+  firstName: string;
+  lastName: string;
+  title?: string;
+  profilePic?: string;
+  description?: string;
+  wage: number;
+  ratings: Array<number>;
+  city: string;
+}
+export interface changeProfile {
```

Comment:
Maybe we can just use one Profile interface and in cases we don't receive a profile, you can just add `profile?: Profile`

Reply:
Interesting, yeah I'll definitely take a look